### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by DevSiroukane** [PHPOffice/PhpSpreadsheet#3794] [Q&A]
Hi.

Please, when will be the next release ? 

I need this fix https://github.com/PHPOffice/PhpSpreadsheet/discussions/3216 **Usage of ezyang/htmlpurifier package**

Regards

**Comment by PowerKiKi** [comment]
[2.0.0](https://github.com/PHPOffice/PhpSpreadsheet/releases/tag/2.0.0) was released
